mkdocs.yaml

```diff
@@ -0,0 +1,6 @@
+site_name: OpenSquirrel
+site_url: http://127.0.0.1:8000/

```

Comment:
This is localhost right? Is that intended?

Reply:
Yes, this is intentional. The local host is also configured by default, but I left it such that we are reminded to update soon as we have a 'official' url.